pkg/exchange/binance/convert.go

```diff
@@ -161,6 +323,29 @@ func toGlobalOrder(binanceOrder *binance.Order, isMargin bool) (*types.Order, er
 	}, nil
 }
 
+func toGlobalFuturesOrder(futuresOrder *futures.Order, isMargin bool) (*types.Order, error) {
+	return &types.Order{
+		SubmitOrder: types.SubmitOrder{
+			ClientOrderID: futuresOrder.ClientOrderID,
+			Symbol:        futuresOrder.Symbol,
+			Side:          toGlobalFuturesSideType(futuresOrder.Side),
+			Type:          toGlobalFuturesOrderType(futuresOrder.Type),
+			Quantity:      util.MustParseFloat(futuresOrder.OrigQuantity),
+			Price:         util.MustParseFloat(futuresOrder.Price),
+			TimeInForce:   string(futuresOrder.TimeInForce),
+		},
+		Exchange:         types.ExchangeBinance,
+		// IsWorking:        futuresOrder.IsWorking,
+		OrderID:          uint64(futuresOrder.OrderID),
+		Status:           toGlobalFuturesOrderStatus(futuresOrder.Status),
+		ExecutedQuantity: util.MustParseFloat(futuresOrder.ExecutedQuantity),
+		CreationTime:     types.Time(millisecondTime(futuresOrder.Time)),
+		UpdateTime:       types.Time(millisecondTime(futuresOrder.UpdateTime)),
+		IsMargin:         isMargin,
+		// IsIsolated:       futuresOrder.IsIsolated,
```

Comment:
do we have other fields from futures?

Reply:
we have STOP_MARKET and STOP_LIMIT in the global type.

might need to add the field `reduceOnly` and `closePosition`

we can add these fields first, and handle it later

Reply:
Understood. Working on it.

Reply:
Added
`exchange.go:submitFuturesOrder() => convert.go:toGlobalFuturesOrder() => types/order.go:SubmitOrder{}`